### ISSUE TITLE
Read/Write internal eeprom

### ISSUE THREAD

**Issue by ziotibia81** [denandz/5am_util#2]
Hi! 
IAW5AM has firmware stored in flash, data handled by this utility I think, and an eeprom where ECU store Immobilizer data and troubles snapshots. There are a way to read an write this eeprom using diag K-Line, I can find exactly how to do this. Are you interested to add support for this?

**Comment by denandz** [comment]
Definitely, if you can provide the specs on reading the eeprom over k-line then that would be a good addition to 5am_util

**Comment by ziotibia81** [comment]
I just found out that IAW5AMEEPROMTool from [here](https://www.von-der-salierburg.de/download/GuzziDiag/) already does this.
But I'm looking for more protocol info.

More info maybe be [here](http://forum.multiecuscan.net/viewtopic.php?t=521&start=10)
